protected/commands/SmokeTestCommand.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Create and remove smoke test data to allow end to end non-destructive testing of FUW workflow on cloud environments
+ *
+ * TODO: make the test id a constant
+ *
+ * @author Rija Menage <rija+git@cinecinetique.com>
+ * @license GPL-3.0
+ */
+class SmokeTestCommand extends CConsoleCommand {
+	
+	public function getHelp() {
+		$helpText = "Create and remove smoke test data to allow end to end non-destructive testing of FUW workflow on cloud environments".PHP_EOL;
+		$helpText .= "Usage: ./protected/yiic smoketest createdata".PHP_EOL;
+		$helpText .= "Usage: ./protected/yiic smoketest removedata".PHP_EOL;
+        return $helpText;
```

Comment:
To include `$helpText .= "Usage: ./protected/yiic smoketest resetdata".PHP_EOL;` as well?

Reply:
Thanks. Updated in rija/gigadb-website@007b7197